### PR TITLE
Update to Gradle 5.5.1

### PR DESCRIPTION
**Pull request by larsgrefer**

### CODE REVIEW

File: gradle/wrapper/gradle-wrapper.properties [hunk 1]
```diff
@@ -1,5 +1,5 @@
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-5.3.1-bin.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-5.5-all.zip
```
Comment (jzheaux) [inline gradle/wrapper/gradle-wrapper.properties]: @larsgrefer what is your reasoning for moving to `-all` from `-bin`?

Comment (larsgrefer) [reply gradle/wrapper/gradle-wrapper.properties]: With the `all` distribution, my IDE (IntelliJ IDEA) can provide better auto-completion and documentation for build scripts. Therefore I have `all` as default on my systems.

Comment (larsgrefer) [reply gradle/wrapper/gradle-wrapper.properties]: @jzheaux Should I change it back to `bin`?

Comment (jzheaux) [reply gradle/wrapper/gradle-wrapper.properties]: Yes, let's please change it back to `bin` since this ticket is just about updating to Gradle 5.5.

Comment (larsgrefer) [reply gradle/wrapper/gradle-wrapper.properties]: Done

**Comment by larsgrefer** [comment]
I've updated the PR to Gradle 5.5.1, which was released in the meantime

**Comment by jzheaux** [comment]
Thanks, @larsgrefer, for another PR! This is now merged into `master`.

**Pull request merged**